spacy/tests/pipeline/test_spancat.py

```diff
@@ -71,6 +71,57 @@ def test_explicit_labels():
     assert spancat.labels == ("PERSON", "LOC")
 
 
+@pytest.mark.parametrize(
+    "max_positive,nr_results", [(None, 4), (1, 2), (2, 3), (3, 4), (4, 4)]
+)
+def test_make_spangroup(max_positive, nr_results):
+    import numpy
+
```

Comment:
Is there a particular reason for doing the imports like this? Can we make it more like the rest of the test suite with the imports at the top like the other numpy testing utils?

Reply:
Definitely!